### DISCUSSION TITLE
Software and Hardware Dimming not working

### DISCUSSION THREAD

**Discussion by mehapps** [MonitorControl/MonitorControl#1387] [Q&A] [answered]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

I plugged in a new display (Lenovo 4434-HE1) to my Mac with a Thunderbolt 2 to DVI-D cable. The same cable works fine to control my BenQ monitor, but doesn't control the Lenovo. I thought: No problem, I'll just use software dimming. However, software dimming is not working on either monitor.

### Expected behavior

Software dimming to work as intended.

### Anything else?

I am using a Mac with OCLP, don't know if that has anything to do with it.

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.3.1
- Mac model: Mac mini (Mid 2011)
- MonitorControl version: 4.1.0 build 7034
- Monitor(s): BenQ RL2455 and Lenovo 4434-HE1
- Monitor Cable(s)/Connection(s): HDMI to BenQ and Thunderbolt 2 to DVI to Lenovo
- Apple Silicon/M1 (yes or no): No
```

**Comment by waydabber** [comment, marked as answer]
How did you try enabling software dimming? it might be that the patched GPU drivers do not support gamma table manipulation properly. In this case you can try disabling that as well for overlay dimming which should work as a last resort.

**Reply by mehapps** [reply]
I no longer have the second monitor, but thanks for the info! Will try if I get a new one
